provisioning/chain.pem

```diff
@@ -0,0 +1,66 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
We are missing `root.crt` file which is required to build Fobnail

Reply:
applied